app/routers/user.router.js

```diff
@@ -9,6 +15,7 @@ const router = express.Router();
 router.get('/me', getUser);
 router.get('/user', getAll);
 router.get('/user/:id', getById);
+router.post('/user/:id', removeUser);
```

Comment:
Why are you using `post` http method for deleting ? I think we should use `delete` for this purpose